Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/mapping/CoronaTestConfigMapper.kt

```diff
@@ -31,7 +31,7 @@ class CoronaTestConfigMapper @Inject constructor() : CoronaTestConfig.Mapper {
                 hoursToDeemTestOutdated = Duration.standardHours(
                     coronaRapidAntigenTestParameters.hoursToDeemTestOutdated.toLong()
                 ),
-                hoursSinceSampleCollectionToShowRiskCard = Duration.standardHours(
+                hoursToShowRiskCard = Duration.standardHours(
```

Comment:
can you call it timeToShow... as it is a Duration (hours have been resolved)

Reply:
renamed to `duration*`